docs/events/intro.md

```diff
@@ -84,3 +86,17 @@ Sub Events
 ----------
 
 Many events have different variations of themselves, these can be different but all based around one common factor (e.g. `PlayerEvent`) or can be an event that has multiple phases (e.g. `PotionBrewEvent`). Take note that if you listen to the parent event class, you will receive calls to your method for *all* subclasses.
+
+Mod Event Bus
+-------------
+
+The mod event bus is primarily meant for listening to lifecycle events in which mods should initialize. Many of these events are also ran in parallel so mods can be initialized at the same time. This does mean you can't directly execute code from other mods in these events, use the `InterModEnqueueEvent` and `InterModProcessEvent` events for that.
```

Comment:
The mod event bus is used for all events that are only called during the load process. This includes lifecycle events yes, but also registry events, and other misc load-time events like color handlers and model bake etc.

Reply:
Also perhaps mention DeferredWorkQueue